pkg/kademlia/dialer.go

```diff
@@ -47,29 +48,37 @@ func (dialer *Dialer) Close() error {
 }
 
 // Lookup queries ask about find, and also sends information about self.
-func (dialer *Dialer) Lookup(ctx context.Context, self pb.Node, ask pb.Node, find pb.Node) (_ []*pb.Node, err error) {
+// If self is nil, pingback will be false.
+func (dialer *Dialer) Lookup(ctx context.Context, self *pb.Node, ask pb.Node, find storj.NodeID, limit int) (_ []*pb.Node, err error) {
```

Comment:
Just curious, why is the Node slice an anonymous variable?

Reply:
during the monkit flurry of additions, i needed a way of naming the `err` return value without doing much else to the semantics of the function. go won't let me just name one return value without naming all of them. we could name the node slice. that would probably be better